src/docfx/build/metadata/UserMetadata.cs

```diff
@@ -24,6 +25,10 @@ internal class InputMetadata
         [JsonProperty("_tocRel")]
         public string TocRel { get; set; }
 
+        [JsonProperty("_op_documentIdPathDepotMapping")]
+        public Dictionary<string, DocumentIdDepotMapping> DocumentIdDepotMapping { get; set; }
```

Comment:
>DocumentIdDepotMapping  [](start = 58, length = 23)

what does this server for?

Reply:
https://dev.azure.com/ceapex/Engineering/_git/Docs.Build?path=%2Fsrc%2FConfigMigration%2Fsrc%2Fmigration%2Fmigrator%2Fdocument-id-migrator.js&version=GBdevelop&line=22&lineEnd=22&lineStartColumn=1&lineEndColumn=122&lineStyle=plain

Reply:
thought you've already read it into GlobalMetadata?

---
In reply to: [343986084](https://github.com/dotnet/docfx/pull/5260#discussion_r343986084) [](ancestors = 343986084)